docs/source/basics/tokenizers.rst

```diff
@@ -0,0 +1,239 @@
+.. _tokenizers_usage_label:
+
+==========
+Tokenizers
+==========
+
+Tokenizers are a key component of any LLM. They convert raw text into token IDs, which index into embedding vectors that are
+understood by the model.
+
+In torchtune, tokenizers play the role of converting Messages object into token IDs and any necessary model-specific special tokens.
+They use byte-pair encoding algorithms to convert raw strings into integer IDs. These are usually based on SentencePiece or TikToken.
```

Comment:
This I'm not sure is super relevant here tbh. Like people prob won't care about the underlying algo (though probably is worth mentioning that SentencePiece and TikToken are the two that we currently support)